base/string.jl

```diff
@@ -1645,6 +1663,13 @@ function search(a::ByteArray, b::Union(Int8,UInt8), i::Integer)
     q = ccall(:memchr, Ptr{UInt8}, (Ptr{UInt8}, Int32, Csize_t), p+i-1, b, n-i+1)
     q == C_NULL ? 0 : Int(q-p+1)
 end
+function search(a::Vector{UInt8}, b::Char, i::Integer)
```

Comment:
Isn't this completely covered by the `search(a::ByteArray, b::Char, i::Integer)` method below?